rspec-rails.gemspec

```diff
@@ -21,7 +22,7 @@ Gem::Specification.new do |s|
     'source_code_uri'   => 'https://github.com/rspec/rspec-rails',
   }
 
-  s.files            = `git ls-files -- lib/*`.split("\n")
+  s.files            = Rake::FileList["lib/**/{.*,*}"].exclude(*File.read(".gitignore").split).reject { |f| File.directory?(f) }
```

Comment:
Is rejection of directories necessary? Will the resulting gem contain those dirs as as empty dirs, or their files as well? Or will those dirs be skipped?

Reply:
When you use `git ls-files`, it skips the empty directories. `.reject` does the same thing since `Rake::FileList` and `Dir` are not capable of this. I personally just feel `Dir` is sufficient alone (without `.exclude` and `.reject`). What do you think?